packages/keyring-controller/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [6.1.0]
```

Comment:
What do you think about using 6.0.1 instead of 6.1.0? It doesn't seem like this adds any explicit features, it's just a general upgrade.

Reply:
Hmm. I could see it either way. A dependency bump certainly isn't a bug fix - I could see someone updating to get it, similar to how one might update to get a feature. It's kindof a feature for us actually, since it helps reduce the extension bundle size.

Reply:
Though this official answer seems to lean more towards it being a patch: https://semver.org/#what-should-i-do-if-i-update-my-own-dependencies-without-changing-the-public-api

Reply:
Hmm, okay. I'll stick with it, then.